src/Microsoft.Health.Fhir.Liquid.Converter/Filters/CollectionFilters.cs

```diff
@@ -67,6 +67,16 @@ public static object GetIndex(object[] collection, int index)
             return null;
         }
 
+        public static object Slice(string input, int offset, int length)
+        {
+            if (length == -1)
+            {
+                length = input.ToString().Length;
```

Comment:
What does the .ToString() do here?

Reply:
Good point, I think this has some mistakes.